Lower system memory requirements for tests to support ci runners <4GiB

diff --git a/crates/arti/src/arti-example-config.toml b/crates/arti/src/arti-example-config.toml
index 16e2d6ac2..33e9870c5 100644
--- a/crates/arti/src/arti-example-config.toml
+++ b/crates/arti/src/arti-example-config.toml
@@ -451,11 +451,11 @@
 # The default is unlimited.
 #
 # Maximum memory use, after which reclamation starts:
-#    memory.max = "8 GiB"
+#    memory.max = "3 GiB"
 # (If anything is specified in `[system.memory]`, this value is mandatory.)
 #
 # When reclaiming memory, we stop when we reach this amount:
-#    memory.low_water = "6 GiB"
+#    memory.low_water = "2 GiB"
 # (The default is 3/4 of `system.memory.max`.)
 
 ##### ONION SERVICES
